omnibus-optimism-goerli.toml

```diff
@@ -48,6 +49,12 @@ defaultValue = "1"
 [setting.sc_pool_owner]
 defaultValue = "0x48914229deDd5A9922f44441ffCCfC2Cb7856Ee9"
 
+[setting.snx_pool_id]
+defaultValue = "2"
```

Comment:
Some networks may have pool taken just because it's the next pool Id. Maybe default to something a little more unique?